kart/reflink_util.py

```diff
@@ -0,0 +1,40 @@
+import shutil
+
+import reflink as rl
+
+
+def reflink(from_, to):
+    """Same as reflink.reflink, but can be called with pathlib.Path objects."""
+    rl.reflink(str(from_), str(to))
+
+
+def try_reflink(*args):
```

Comment:
this thing's a bit wacky? IMO callers should just call two different functions, rather than one function that does two different things.

Reply:
You're right. Done